h2drivers/src/main/java/org/h2gis/drivers/asciiGrid/AsciiGridDriverFunction.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * H2GIS is a library that brings spatial support to the H2 Database Engine
+ * <http://www.h2database.com>.
+ *
+ * H2GIS is distributed under GPL 3 license. It is produced by CNRS
+ * <http://www.cnrs.fr/>.
+ *
+ * H2GIS is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * H2GIS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * H2GIS. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * For more information, please consult: <http://www.h2gis.org/>
+ * or contact directly: info_at_h2gis.org
+ */
+package org.h2gis.drivers.asciiGrid;
+
+import java.io.File;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import org.h2gis.drivers.utility.FileUtil;
+import org.h2gis.h2spatialapi.DriverFunction;
+import org.h2gis.h2spatialapi.ProgressVisitor;
+
+/**
+ * A driver to import and export Ascii grid image
+ * 
+ * @author Erwan Bocher
+ */
+public class AsciiGridDriverFunction implements DriverFunction{
```

Comment:
You never register this driver. You should check for availability of dependency, and if available then register this driver in OSGi.